charts/zookeeper-operator/templates/clusterrole.yaml

```diff
@@ -13,15 +13,13 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - nodes
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
   verbs:
-  - get
-  - watch
-  - list
-- apiGroups:
-  - admissionregistration.k8s.io
-  resources:
   - "*"
-  verbs:
-  - '*'
```

Comment:
@anishakj would it make sense to first integrate https://github.com/pravega/zookeeper-operator/pull/133 and then rebase this patch on top of that?